examples/tutorials/nb_python/Habitat2_Quickstart.py

```diff
@@ -339,7 +336,7 @@ def update_metric(self, *args, episode, task, observations, **kwargs):
 # %% [markdown]
 # We now add all the previously defined task, sensor, and measurement
 # definitions to a config file to finish defining the new Habitat task. For
-# examples of more configs [see here](https://github.com/facebookresearch/habitat-lab/tree/main/configs/tasks/rearrange).
+# examples of more configs [see here](https://github.com/facebookresearch/habitat-lab/tree/main/tasks/rearrange).
```

Comment:
I guess it should be this link https://github.com/facebookresearch/habitat-lab/tree/main/habitat-lab/habitat/config/tasks/rearrange